component/pyroscope/java/java_stub.go

```diff
@@ -0,0 +1,37 @@
+//go:build (linux && !(amd64 || arm64)) || !linux
+
+package java
+
+import (
+	"context"
+
+	"github.com/grafana/agent/component"
+	"github.com/grafana/agent/pkg/flow/logging/level"
+)
+
+func init() {
+	component.Register(component.Registration{
+		Name: "pyroscope.java",
+		Args: Arguments{},
+
+		Build: func(opts component.Options, args component.Arguments) (component.Component, error) {
+			level.Warn(opts.Logger).Log("msg", "the pyroscope.java component only works on linux; enabling it otherwise will do nothing")
```

Comment:
```suggestion
			level.Warn(opts.Logger).Log("msg", "the pyroscope.java component only works on linux for amd64 and arm64; enabling it otherwise will do nothing")
```